RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ResourcesList+View.swift

```diff
@@ -92,7 +92,7 @@ extension ResourcesList.View {
 						Text($0.text)
 					}
 				}
-				.pickerStyleRadixSegmented()
+				.pickerStyle(.segmented)
```

Comment:
Don't we want to style it anymore? Or did you find another way?

Reply:
Matt did not want it and wanted the same between picker in `ThirdPartyDeposits/ResourceList` and in `ImportMnemonic`